src/Raven.Server/ServerWide/Commands/PutDatabaseClientConfigurationCommand.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Amqp.Framing;
+using Raven.Client.Documents.Indexes;
+using Raven.Client.Documents.Operations.Configuration;
+using Raven.Client.ServerWide;
+using Raven.Server.Extensions;
+using Raven.Server.Rachis;
+using Raven.Server.Utils;
+using Sparrow;
+using Sparrow.Json.Parsing;
+using Index = Raven.Server.Documents.Indexes.Index;
+
+namespace Raven.Server.ServerWide.Commands
+{
+    public class PutDatabaseClientConfigurationCommand : UpdateDatabaseCommand
+    {
+
+        public ClientConfiguration Client;
```

Comment:
let's rename `Client` to `Configuration`

Reply:
Done